sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -74,6 +74,7 @@ export interface EventDataBatch {
 
 // @public
 export interface EventHubClientOptions {
+    customEndpointAddress?: string;
```

Comment:
Would we be adding similar support for service-bus?

Reply:
Potentially (though the request and API review we had was specifically for event hubs.) For now I'd consider that out of scope.

Reply:
I agree with @chradek here